Gopkg.toml

```diff
@@ -1,8 +1,8 @@
 [[constraint]]
   name = "github.com/ibm-messaging/mq-golang"
-  version = "2.0.0"
+  version = "3.0.0"
```

Comment:
There is no version 3.0.0 of the github.com/ibm-messaging/mq-golang repo so this will break people using version control. 

Did you want it to use the latest of mq-golang or do we need to release a new version?

Reply:
You should also be seeing a PR for go-lang and once that's merged, I'm going to create v3.0.0 from it